examples/android/minimal_kts/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Aug 17 12:18:19 CEST 2022
+#Wed Feb 08 13:19:59 CET 2023
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
I think this is desirable for Android, because the examples do not work with all Gradle versions, and I would like the local launch of the examples to be as simple as possible